eos-updater-flatpak-installer/eos-updater-flatpak-installer.c

```diff
@@ -0,0 +1,262 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
```

Comment:
This is going to require some sort of service file that starts it at boot after an upgrade.